ddtrace/llmobs/_telemetry.py

```diff
@@ -156,6 +157,16 @@ def record_llmobs_annotate(span: Optional[Span], error: Optional[str]):
     )
 
 
+def record_llmobs_user_processor_called(error: bool) -> None:
+    tags = [("error", "1" if error else "0")]
+    telemetry_writer.add_count_metric(
+        namespace=TELEMETRY_NAMESPACE.MLOBS,
+        name=LLMObsTelemetryMetrics.USER_PROCESSOR_CALLED,
```

Comment:
We'll need to add a PR to add this metric to the allow list (ex: https://github.com/DataDog/dd-go/pull/173143)!